app/build.gradle

```diff
@@ -75,6 +75,11 @@ android {
     buildFeatures {
         dataBinding true
         viewBinding true
+        compose true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion '1.1.1'
```

Comment:
You can set this to your `composeUIVersion` variable so you don't have to update it later!